lib/diagnostic.rb

```diff
@@ -26,26 +26,36 @@
 # be writable.
 
 ##
-# your response here
+class Person
+  attr_reader :name, :age
+  attr_writer :name, :location
+end
 ##
 
 ##
 # Instatiate a new Person object using arguments "Dave", 32, and "Ohio".
 # Then, in the next line, change this Person's location property to
 # "Somerville". Finally, assign the modified Person to `Response.dave` below.
-
+def initialize (Dave,32, Ohio)
```

Comment:
We want to instantiate an instance of `Person` with the three given initial values and then modify its `location`.

```ruby
dave = Person.new('Dave', 32, 'Ohio')
dave.location = 'Somerville'
```